### PR TITLE
Chef 13 compatibility fixes

### PR DESCRIPTION
**Pull request by nhcj**

### CODE REVIEW

**Comment by nhcj** [comment]
[Chef 13: yum-kernel-osuosl](https://trello.com/c/d2uLqG4K/903-chef-13-yum-kernel-osuosl)

File: .kitchen.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 verifier:
   name: inspec
 
+transport:
+  name: ssh
+
```
Comment (TravisWhitehead) [inline .kitchen.yml:7]: Why is this needed?

**Review by TravisWhitehead — CHANGES_REQUESTED** [review]

File: .kitchen.yml (hunk 1 above)
Comment (nhcj) [reply .kitchen.yml:7]: The rsync transport doesn’t support Inspec, apparently.

Comment (TravisWhitehead) [reply .kitchen.yml:7]: ~~I ran a `kitchen test` without specifying the transport, and I didn't see any issues. What problems were you running into?~~

Comment (TravisWhitehead) [reply .kitchen.yml:7]: Just kidding-- Seems problems only appear on `blue`, which I forgot to run it on the first time around.

**Review by TravisWhitehead — APPROVED** [review]

**Review by kindlehl — APPROVED** [review]
Tests pass, no deprecation warnings :^) Good job!

File: attributes/default.rb [hunk 2]
```diff
@@ -3,11 +3,9 @@
...
 default['yum']['kernel-osuosl']['gpgkey'] = 'https://ftp.osuosl.org/pub/osl/repos/yum/RPM-GPG-KEY-osuosl'
-case node['kernel']['machine']
-when 'ppc64le'
-  default['yum']['kernel-osuosl']['baseurl'] =
-    'https://ftp.osuosl.org/pub/osl/repos/yum/openpower/centos-$releasever/ppc64le/kernel-osuosl/'
-else
-  default['yum']['kernel-osuosl']['baseurl'] =
-    'https://ftp.osuosl.org/pub/osl/repos/yum/$releasever/kernel-osuosl/$basearch/'
-end
+default['yum']['kernel-osuosl']['baseurl'] = case node['kernel']['machine']
```
Comment (ramereth) [inline attributes/default.rb]: Can you bring the case statement down one line and then indent it? That should still make cookstyle happy.

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -5,23 +5,23 @@
...
   platform: 'debian',
-  version: '8.4'
+  version: '8.4',
 }.freeze
 
 ALL_PLATFORMS = [
   CENTOS_6,
   CENTOS_7,
-  DEBIAN_8
+  DEBIAN_8,
```
Comment (ramereth) [inline spec/spec_helper.rb]: This only supports CentOS 7 so you can go ahead and remove C6 and Debian.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by TravisWhitehead — APPROVED** [review]

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -5,23 +5,21 @@
...
   platform: 'centos',
-  version: '6.7'
+  version: '6.7',
 }.freeze
 
 DEBIAN_8 = {
   platform: 'debian',
-  version: '8.4'
+  version: '8.4',
 }.freeze
```
Comment (ramereth) [inline spec/spec_helper.rb]: Please remove this.

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -5,23 +5,21 @@
...
   platform: 'centos',
-  version: '7.2.1511'
+  version: '7.2.1511',
 }.freeze
 
 CENTOS_6 = {
   platform: 'centos',
-  version: '6.7'
+  version: '6.7',
 }.freeze
```
Comment (ramereth) [inline spec/spec_helper.rb]: Please remove this.

**Review by ramereth — APPROVED** [review]

**Comment by nhcj** [comment]
!bump major *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v2.0.0.  Have a nice day!
